task 1 lesson 1

MCaberGIT opened the pull request.

MCaberGIT committed:
task 1 lesson 1

MCaberGIT committed:
task 1 lesson 1

patryk-stepniak reviewed (changes requested):
O wiele lepiej :)
Brakuje jednak konfiguracji checkstylera w build.gradle:
checkstyle {
    showViolations = true
    ignoreFailures = false
    maxWarnings = 0
    maxErrors = 0
}

MCaberGIT committed:
dodano konfiguracje check style w build.gradle
 checkstyle {
        showViolations = true
        ignoreFailures = false
        maxWarnings = 0
        maxErrors = 0
    }

MCaberGIT committed:
lesson1 task 1 - fix config
